README.md

```diff
@@ -85,78 +91,128 @@ PgJDBC uses java.util.logging for logging.
+
+#### Java System Properties
+The following  properties can be set.
+
+| Property                         | Type | Default | Maps to       | Description                                                                                                                                           |
+|----------------------------------|------|---------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| org.postgresql.pguser            |      |         | user          | same as the user connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).                  |
+| org.postgresql.pgpassword        |      |         | password      | same as the password connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).              |
+| org.postgresql.pghost            |      |         | host          | same as the host connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).                  |
+| org.postgresql.pgport            |      |         | port          | same as the port connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).                  |
+| org.postgresql.pgdatabase        |      |         | dbname        | same as the dbname connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).                |
+| org.postgresql.pgservice         |      |         | service       | same as the service connection parameter. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html).               |
+| org.postgresql.pgpassfile        |      |         | passfile      | same as the passfile connection parameter. [The Password File](https://www.postgresql.org/docs/current/libpq-pgpass.html).                            |
+| org.postgresql.pgservicefile     |      |         | PGSERVICEFILE | name of the per-user connection service file. [The Connection Service File](https://www.postgresql.org/docs/current/libpq-pgservice.html).            |
+| org.postgresql.pgsysconfdir      |      |         | PGSYSCONFDIR  | sets the directory containing the pg_service.conf file. See also: [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html). |
+| pgjdbc.config.cleanup.thread.ttl | long | 30000   |               | The driver has an internal cleanup thread which monitors and cleans up unclosed connections. This property sets the duration (in milliseconds) the cleanup thread will keep running if there is nothing to clean up. |
+
+#### Operating System Environment variables
+The following environment variables can be set.
```

Comment:
it is about improving compatibility with libpq
> What does "env vars can be set" mean? 
```
# An example of setting environment variables:
export PGUSER=my_user
export PGPASSWORD=my_password
```
> What does it give to the end users?
```
# then jdbc and libpq-based programs can be launched identically:
java ... MainJdbcBasedProgram
./main_libpq_based_program
```

Reply:
What I mean is that it would be better to phrase the features in terms of their usability.
The fact that environment variable can be set does not provide much to the end users.

It might be better to have something like "Some of the settings could be passed from environment variables as follows:..."

Reply:
Is it good enough: https://github.com/pgjdbc/pgjdbc/pull/2646/commits/3db232a53641c86c8183cbf6b88caf4cb503cc71